### PR TITLE
DRUP-1043: #ready adds menu title, adds css 'mobile-collapsible'

### PR DESCRIPTION
**Pull request by emb03**
@z3cka I completely deleted module from my local and replaced with downloaded feature from prod, I am not sure why the views default inc file is deleting things. I will annotate the code.

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc [hunk 1]
```diff
@@ -267,36 +267,6 @@ function uclalib_bookplates_views_default_views() {
   );
```
Comment (emb03) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc]: Not sure what is happening here, it's deleting something. I did not touch views on prod.

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc [hunk 2]
```diff
@@ -267,36 +267,6 @@ function uclalib_bookplates_views_default_views() {
...
   $handler->display->display_options['row_options']['hide_empty'] = TRUE;
   $handler->display->display_options['defaults']['row_options'] = FALSE;
-  $handler->display->display_options['defaults']['relationships'] = FALSE;
```
Comment (emb03) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc]: Not sure what is happening here, it's deleting something. I did not touch views on prod.

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc [hunk 3]
```diff
@@ -267,36 +267,6 @@ function uclalib_bookplates_views_default_views() {
   );
   $handler->display->display_options['row_options']['hide_empty'] = TRUE;
   $handler->display->display_options['defaults']['row_options'] = FALSE;
```
Comment (emb03) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc]: Not sure what is happening here, it's deleting something. I did not touch views on prod.

**Comment by z3cka** [comment]
When rolling from Prod (or staging) the base branch should be staging, not master.

**Comment by emb03** [comment]
K, I will do this on Monday. Thx!

**Pull request closed without merging**